fontique/Cargo.toml

```diff
@@ -28,16 +28,17 @@ system = ["std"]
 skrifa = { workspace = true }
 peniko = { workspace = true }
 smallvec = "1.13.2"
-memmap2 = { version = "0.9.4", optional = true }
-unicode-script = { version = "0.5.6", optional = true }
+memmap2 = { version = "0.9.5", optional = true }
+unicode-script = { version = "0.5.7", optional = true }
 core_maths = { version = "0.1.0", optional = true }
 icu_properties = { version = "1.5.1", optional = true }
 icu_locid = "1.5.0"
 hashbrown = "0.14.5"
 
 [target.'cfg(target_family="windows")'.dependencies]
-windows = { version = "0.58.0", features = ["implement", "Win32_Graphics_DirectWrite"] }
-windows-core = { version = "0.58" }
+dwrote = "0.11.1"
+winapi = { version = "0.3.9", features = ["dwrite", "dwrite_1", "dwrite_3", "winnt", "unknwnbase", "libloaderapi", "winnls"] }
+wio = "0.2.2"
```

Comment:
This isn't right. Should be reverted and potentially windows version bumped if required.